data/rapid_config.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
This file has been removed from source control, but after an 'npm build' or 'npm build-internal' it would be a jsonified version of whatever is inside data/rapid_config.yaml's "public" or "internal" data block, respectively.